src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -41,9 +46,61 @@ def post(self, request: Request) -> Response:
         Write issue occurrence and event data to a Kafka topic
         ``````````````````````````````````````````````````````
         :auth: superuser required
+        :pparam: string dummyEvent: pass 'True' to load a dummy event instead of providing one in the request
+        :pparam: string dummyOccurrence: pass 'True' to load a dummy occurrence instead of providing one in the request
         """
-        event = request.data.pop("event")
-        occurrence = request.data
+        event = {}
+        if request.query_params.get("dummyEvent") == "True":
+            user = User.objects.get(id=request.user.id)
+            projects = user.get_projects()
+            if not projects:
+                return Response(
+                    "Requesting user must belong to at least one project.",
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+            event = {
+                "event_id": "44f1419e73884cd2b45c79918f4b6dc4",
+                "project_id": projects[0].id,
+                "title": "This is bad",
+                "platform": "python",
+                "tags": {"environment": "prod"},
+                "timestamp": ensure_aware(datetime.now()),
+                "message_timestamp": ensure_aware(datetime.now()),
+            }
+        else:
+            event = request.data.pop("event", None)
+
+        if not event:
+            return Response(
+                "Must pass an event or query param of dummyEvent=True",
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        occurrence = {}
+        if request.query_params.get("dummyOccurrence") == "True":
+            occurrence = {
+                "fingerprint": ["some-fingerprint"],
+                "issue_title": "something bad happened",
+                "subtitle": "it was bad",
+                "resource_id": "1234",
+                "evidence_data": {"Test": 123},
+                "evidence_display": [
+                    ("Attention", "Very important information!!!", True),
+                    ("Evidence 2", "Not important", False),
+                    ("Evidence 3", "Nobody cares about this", False),
```

Comment:
i think this is a dict now not a tuple
https://github.com/getsentry/sentry/blob/0f6923d5dc11e8552e6620dd14596f1d4efe630c/src/sentry/issues/issue_occurrence.py#L113